stores/user.ts

```
@@ -2,14 +2,16 @@ import { defineStore } from "pinia";
 import { getQuestionsFromContext, type Question } from "~/utils/questions";
+import type { IntegrationPayload } from "~/utils/integrations";
+import type { IntegrationSlug } from "~/server/models/formResponse";
 
 export const useUserStore = defineStore("user", {
   state: () => ({
     user: null as CurrentUserData | null,
     token: null as string | null,
     formResponse: null as any, // To be replaced with a specific type if available
     donationIntent: null as "today" | "soon" | null,
-    loadingLogin: true as boolean,
+    loadingLogin: true as Boolean,
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)